src/main/Game.java

```diff
@@ -0,0 +1,97 @@
+import Board.Board;
+import Board.StandardBoard;
+import Enums.Result;
+import Player.Player;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import Player.Player;
+import Player.HumanPlayer;
+import Enums.Colors;
+
+public class Game {
+
+    private Board gameBoard;
+    private Map<Player, List<Move> >  playedMoves;
+    private Player turn;
+    private Player player1;
+    private Player player2;
+    private Map<Player, Colors> playerColor;
+    private Map<Player, Result> result;
+
+    public Game()
+    {
+        gameBoard = new StandardBoard();
+        playedMoves = new HashMap<Player, List<Move>>();
+        playerColor = new HashMap<Player, Colors>();
+        result = new HashMap<Player, Result>();
+    }
+    public Board getGameBoard() {
+        return gameBoard;
+    }
+
+    public Map<Player, List<Move>> getPlayedMoves() {
+        return playedMoves;
+    }
+
+    public Player getTurn() {
+        return turn;
+    }
+
+    public Player getPlayer1() {
+        return player1;
+    }
+
+    public Player getPlayer2() {
+        return player2;
+    }
+
+    public Map<Player, Colors> getPlayerColor() {
+        return playerColor;
+    }
+
+    public Map<Player, Result> getResult() {
+        return result;
+    }
+
+    public void setGameBoard(Board gameBoard) {
+        this.gameBoard = gameBoard;
+    }
+
+    public void setPlayedMoves(Map<Player, List<Move>> playedMoves) {
+        this.playedMoves = playedMoves;
+    }
+
+    public void setTurn(Player turn) {
+        this.turn = turn;
+    }
+
+    public void setPlayer1(Player player1) {
+        this.player1 = player1;
+    }
+
+    public void setPlayer2(Player player2) {
+        this.player2 = player2;
+    }
+
+    public void setPlayerColor(Map<Player, Colors> playerColor) {
+        this.playerColor = playerColor;
+    }
+
+    public void setResult(Map<Player, Result> result) {
+        this.result = result;
+    }
```

Comment:
I question the need for these setters. I think we can pass a lot of the things like player 1 and player2 via the constructor and get rid of the setters since they don't change. There are a couple other fields that don't change and don't need setters. One that does need setter is Turn, since that changes every turn